src/main/java/server/RequestBuilder.java

```diff
@@ -88,7 +88,7 @@ public void setBody() {
     public void setResourceIdentifier() {
         String[] splitPath = message.split(" ")[1].split("(?=/)");
         if (splitPath.length == 1) {
-            this.resourceIdentifier = "";
+            this.resourceIdentifier = "index.html";
         } else {
             this.resourceIdentifier = message.split(" ")[1].split("(?=/)")[1].split("/")[1];
         }
```

Comment:
Do you do any splitting like this anywhere else? If so it might be nice to have this splitting logic as a separate class that deals with parsing urls.

Reply:
went down this road, but decided to hold off making a separate class for now